src/common/pp-ga/types.ts

```diff
@@ -12,4 +12,5 @@ export const GACustomFieldsIndex = {
   annotationId: 'dimension6',
   annotationLink: 'dimension7',
   isQuoteBlank: 'dimension8',
+  annotationRequestId: 'dimension9',
```

Comment:
This has to get defined in GA settings as well. Easy definition but need to be done for the new field to appear in GA data views.

Reply:
Ok, thanks for reminding